eth_portal/bridge/run.py

```diff
@@ -120,3 +124,29 @@ def load_private_keys():
             sys.exit(INVALID_KEY_ENV_ERROR)
         else:
             return keys
+
+
+def load_provider(provider_arg):
+    if provider_arg == "custom":
+        try:
+            client_id = os.environ["AUTH_CLIENT_ID"]
+            client_secret = os.environ["AUTH_CLIENT_SECRET"]
+            client_url = os.environ["AUTH_CLIENT_URL"]
+        except KeyError:
+            sys.exit(INVALID_AUTH_PROVIDER_ENV_ERROR)
+        headers = {
+            "headers": {
+                "Content-Type": "application/json",
+                "CF-Access-Client-Id": client_id,
+                "CF-Access-Client-Secret": client_secret,
```

Comment:
These HTTP headers aren't a global standard. I went and looked up that they are apparently used by Cloudflare. I'm not sure if our ops team is actually using cloudflare access mechanisms or if the header names are copypasta.

Either way, I guess we can call the provider time `cloudflare-auth` instead of `custom`.